index.js

```diff
@@ -137,21 +150,68 @@ function useCard()
     //Play card if valid move, otherwise ignore
     if (isValidCard == true)
     {
+        let cardBeingPlayed = players[gameTurn].playerDeck.getCard(cardIndex);
         alert("Debug: Valid move.");
+        
+        
+        //Will run if there is a stackable card played, +2 or +4
+        if(drawStack.stackAmt != 0){
+                if(cardBeingPlayed.value != drawStack.cardValue){
+                    alert("Card chosen Doesn't stack");
+                    return;
+                }else if(cardBeingPlayed.value == 1 && cardBeingPlayed.color != 'Special'){
+                    alert("Card chosen Doesn't stack");
+                    return;
+                }
+                
+        }
+        
         players[gameTurn].playerDeck.playCard(cardIndex);
-        gameTurn++;
+        
+        if(cardBeingPlayed.color == 'Special'){
+            if(cardBeingPlayed.value == 0){
+                cardWild();
```

Comment:
Issues I found (same as first 2 bullet points I wrote on line 175):
- Next player has no card that is valid to play except another special card (when valid card is attempted to be played, it says Valid Move, but then another popup says the card does not stack)
- If the next player tries to draw a new hand, the screen immediately switches back to other player's hand.

Reply:
Same issue as first